modules/renter.go

```diff
@@ -335,9 +335,16 @@ type Renter interface {
 	// blocking, including downloads of `offset` and `length` type.
 	DownloadAsync(params RenterDownloadParameters) error
 
+	// ClearDownloadHistory clears the download history of the renter.
+	ClearDownloadHistory()
```

Comment:
We try to make sure that new interface methods for the modules return an error and then at the beginning we do:
```
if err := tg.Add(); err != nil {
  return err
}
defer tg.Done()
```